<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCodeGames</class>
 <widget class="QWidget" name="OpenCodeGames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenCodeGames</string>
  </property>
  <widget class="QWidget" name="Menu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Menu{ background-image: url(:/images/Menu.png); }</string>
   </property>
   <widget class="QWidget" name="CenterWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>65</y>
      <width>340</width>
      <height>380</height>
     </rect>
    </property>
    <widget class="QWidget" name="HomeCenterWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>340</width>
       <height>380</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#HomeCenterWidget{ background-image: url(:/images/HomeCenterWidget.png); }</string>
     </property>
     <widget class="QLabel" name="UserNameValue">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>330</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NEWSLabel">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>100</y>
        <width>330</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Hack'; color:#ffffff;&quot;&gt;NEWS&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="StatisticsButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#StatisticsButton{ border-image: url(:/images/StatisticsButton.png); }
#StatisticsButton:pressed{ border-image: url(:/images/StatisticsButtonPressed.png); }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="ProgressWidget" native="true">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>55</y>
        <width>162</width>
        <height>12</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ProgressWidget{ background-image: url(:/images/ProgressBarDeactived.png); }</string>
      </property>
      <widget class="QWidget" name="ProgressBar" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="ProgressBarImage" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>162</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#ProgressBarImage{ background-image: url(:/images/ProgressBarActived.png); }</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="ProgressLevel">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>52</y>
        <width>34</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ProgressValue">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>66</y>
        <width>164</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PlayCenterWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>340</width>
       <height>380</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#PlayCenterWidget{ background-image: url(:/images/PlayCenterWidget.png); }</string>
     </property>
    </widget>
    <widget class="QWidget" name="SettingsCenterWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>340</width>
       <height>380</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#SettingsCenterWidget { background-image: url(:/images/SettingsCenterWidget.png); }</string>
     </property>
     <widget class="QLabel" name="SUsernameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SUsernameValue">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>44</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/SettingsText.png);</string>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="SPasswordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>130</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SPasswordValue">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>124</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/SettingsText.png);</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="SNewPasswordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New
Password :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="SNewPasswordValue">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>190</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/SettingsText.png);</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="SRetypeNewPasswordValue">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>256</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/SettingsText.png);</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="SRetypeNewPasswordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>225</y>
        <width>130</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Retype New
Password :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="SSaveButton">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>328</y>
        <width>116</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#SSaveButton{ border-image: url(:/images/SaveButton.png); }
#SSaveButton:pressed{ border-image: url(:/images/SaveButtonPressed.png); }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="SCancelButton">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>328</y>
        <width>184</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="MenuBar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#MenuBar{ background-image: url(:/images/MenuBar.png); }</string>
    </property>
    <widget class="QPushButton" name="HomeButton">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>5</y>
       <width>69</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#HomeButton{ border-image: url(:/images/HomeButton.png); }
#HomeButton:pressed{ border-image: url(:/images/HomeButtonPressed.png); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PlayButton">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>5</y>
       <width>96</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#PlayButton{ border-image: url(:/images/PlayButton.png); }
#PlayButton:pressed{ border-image: url(:/images/PlayButtonPressed.png); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="QuitButton">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>5</y>
       <width>69</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#QuitButton{ border-image: url(:/images/QuitButton.png); }
#QuitButton:pressed{ border-image: url(:/images/QuitButtonPressed.png); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingsButton">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>5</y>
       <width>96</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#SettingsButton{ border-image: url(:/images/SettingsButton.png); }
#SettingsButton:pressed{ border-image: url(:/images/SettingsButtonPressed.png); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
